### ISSUE TITLE
0.3.0 failed on lapack-test

### ISSUE THREAD

**Issue by xkszltl** [xianyi/OpenBLAS#1574]
System spec: Intel SKL / CentOS 7 / gcc-7

```
make PREFIX=/tmp/scratch/OpenBLAS/install.V54Ovz8vgp/root/usr/local lapack-test -j36
(cd ./lapack-netlib/TESTING && rm -f x* *.out)
make -j 36 -j 1 -C ./lapack-netlib/TESTING xeigtstc  xeigtstd  xeigtsts  xeigtstz  xlintstc  xlintstd  xlintstds  xlintstrfd  xlintstrfz  xlintsts  xlintstz  xlintstzc xlintstrfs xlintstrfc
make[1]: warning: -jN forced in submake: disabling jobserver mode.
make[1]: Entering directory '/tmp/scratch/OpenBLAS/lapack-netlib/TESTING'
make[1]: Leaving directory '/tmp/scratch/OpenBLAS/lapack-netlib/TESTING'
make[1]: *** No rule to make target 'xeigtstc'.  Stop.
make: *** [Makefile:297: lapack-test] Error 2
```

**Comment by brada4** [comment]
First build openblas library by running `make`
Then change to that directory TESTING and run `make -sj`

**Comment by martin-frbg** [comment]
This appears to be an overlooked change from the update to 3.8.0 (or perhaps even earlier), these tests
have been split into separate directories, EIG for xeigtst and  LIN  for xlintst.

**Comment by xkszltl** [comment]
Also got issues with `make blas-test` where `(cd ./lapack-net/BLAS && cat *.out)` reports `*.out` doesn't exist.

**Comment by martin-frbg** [comment]
Should be fixed now through #1579 and #1581
